src/CustomerProfile.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export default function CustomerProfile({ customerProfile }) {
+  return (
+    <div className="cont-profile">
+      <h2>Profile</h2>
+      {customerProfile.length !== 0 ? (
+        <ul>
+          <li>Id: {customerProfile.id}</li>
+          <li>Email: {customerProfile.email}</li>
+          <li>Vip: {customerProfile.vip ? "Yes" : "No"}</li>
+          <li>Phone no: {customerProfile.phoneNumber}</li>
+        </ul>
+      ) : (
+        <></>
+      )}
```

Comment:
here instead of a ternary operator, we can use `&&` (useful for when we don't care about rendering anything for the false condition) for example:
```
{customerProfile.length > 0 && (
//code you want to render
```